components/jaggery-core/org.jaggeryjs.jaggery.core/src/main/java/org/jaggeryjs/jaggery/core/manager/CommonManager.java

```diff
@@ -301,35 +306,47 @@ public static void print(Context cx, Scriptable thisObj, Object[] args, Function
         OutputStream out = (OutputStream) jaggeryContext.getProperty(CommonManager.JAGGERY_OUTPUT_STREAM);
         if (args[0] instanceof StreamHostObject) {
```

Comment:
It is a good practice to use a variable for args[0] since you are accessing args[0] couple of times.